kernelci/config/base.py

```diff
@@ -156,6 +182,14 @@ def match(self, **kw):
         filter_values = tuple(kw.get(k) for k in self._keys)
         return filter_values in self._values
 
+    def combine(self, items):
+        keys = tuple(items['keys'])
+        if keys != self._keys:
+            return False
+
+        self._values.extend(list(tuple(values) for values in items['values']))
```

Comment:
I think we can simply use a list comprehension here

```suggestion
        self._values.extend([tuple(values) for values in items['values']])
```